docs/infra/api-markdown-documenter/render-api-documentation.mjs

```diff
@@ -78,13 +79,42 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 
 	const config = getApiItemTransformationConfigurationWithDefaults({
 		apiModel,
-		documentBoundaries: [
-			ApiItemKind.Class,
-			ApiItemKind.Enum,
-			ApiItemKind.Interface,
-			ApiItemKind.Namespace,
-			ApiItemKind.TypeAlias,
-		],
+		hierarchy: {
+			[ApiItemKind.Model]: HierarchyKind.Document,
+			[ApiItemKind.Namespace]: HierarchyKind.Folder,
+			[ApiItemKind.Package]: HierarchyKind.Folder,
```

Comment:
These values are the defaults for their respective types, but since we override document naming policy for them, seems reasonable to add them here explicitly to make the relationship clear.